cli/src/common/boto3/ec2.py

```diff
@@ -113,6 +113,25 @@ def describe_image_by_name_tag(self, image_name):
         owners = ["self"]
         return self.describe_images(ami_ids=[], filters=filters, owners=owners)[0]
 
+    @AWSExceptionHandler.handle_client_exception
+    def describe_instances_by_ami_id(self, image_id):
+        """Describe instance by ami id."""
+        return [
+            instance.get("InstanceId")
+            for result in self._paginate_results(
+                self._client.describe_instances,
+                Filters=[
+                    {
+                        "Name": "image-id",
+                        "Values": [
+                            image_id,
+                        ],
+                    },
+                ],
```

Comment:
nit: you can save some lines by removing the ending commas, the auto-format should compact it in a single line.

Reply:
done